raspberry/setup.py

```diff
@@ -36,10 +36,7 @@
           exclude=["tests/"]),
       include_package_data=True,
       install_requires=[
-          'smbus', 'pigpio', 'bitarray', 'pyserial', 'VL53L0X', 'pandas', 'numpy', 'matplotlib', 'wheel', 'control'
-      ],
-      dependency_links=[
-          "git+https://github.com/pimoroni/VL53L0X-python.git",
+          'pigpio', 'smbus', 'bitarray', 'pyserial', 'VL53L0X @ git+https://github.com/pimoroni/VL53L0X-python.git', 'pandas', 'numpy', 'matplotlib', 'control'
```

Comment:
Are `matplotlib` and `control` necessary?

Reply:
checked and updated